### ISSUE TITLE
JAWS no longer switches to form mode at tablist

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/standards-support#746]
### Summary

1. Go to https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/
2. navigate with TAB key to the tab

### Expected result

Switch to form mode to be able to navigate between tabs with arrow keys

### Actual result

No change to form mode: tabs cannot be reached with arrow keys

#### JAWS version and build number
JAWS 2023.2307.37

The problem is new. It does not occur with any JAWS version before 2023

#### Operating System and version
Windows 10

#### Browser and version:
Current version of Firefox, Edge and Chrome

**Comment by stevefaulkner** [comment]
thanks @JAWS-test will check it out

**Comment by stevefaulkner** [comment]
@JAWS-test i tested with JAWS 2023.2307.37 and chrome Version 116.0.5845.112 (Official Build) (64-bit) using this test page: https://cdpn.io/pen/debug/VwqLzGV

I found i was able to navigate between tabs using the up/down arrow keys and select a tab using the space bar. Is this not the expected behaviour?

**Comment by JAWS-test** [comment]
@stevefaulkner On all JAWS versions up to and including 2022, as well as all other screen readers, TAB navigation to a tab causes the screen reader to switch to forms mode so that I can use the right or left arrow keys to navigate between tabs. Since version 2023 this is no longer possible, JAWS remains in VPC mode even with TAB navigation (this is exactly the behavior you describe).

**Comment by sslawrence521** [comment]
@stevefaulkner @JAWS-test I do a lot of testing on the ServiceNow platform and have been tearing my hair out trying to figure out why all of sudden JAWS stopped going into forms mode on forms as well as tablists. I thought it was something that changed in the app. I created bugs for it but wasn't sure if they were valid since you can manually enter forms mode and I assumed native screen reader users would know this and it was really just testers like me that were confounded. I hope this is going to be fixed soon.

**Comment by stevefaulkner** [comment]
@BrettLewisVispero @ggordon-vispero @RFischer-FS   do any of you have any thoughts on this?

**Comment by sslawrence521** [comment]
@stevefaulkner I created a support ticket with Vispero regarding this behavior and apparently it's a bug (14525).

**Comment by electrickite** [comment]
I ran into this issue debugging a tab widget just today with JAWS 2024. I ended up testing the same W3C tab example in the issue description and was surprised to find that JAWS was not entering pass through mode when moving focus to the tab control. I can press Enter on the tab, which seems to switch JAWS into forms mode and then arrow keys and the rest of the keyboard interaction works as expected and documented on the APG site. Note that this same behavior occurs in Chrome, Firefox, and Edge on Windows 10.

NVDA with the same example site and same browsers behaves as expected.

JAWS: 2024.2312.53
Firefox: 122.0
Chrome: 121.0
Windows: 19045.3930
